packages/contracts/contracts/libs/v0.8.x/minter-libs/MaxInvocationsLib.sol

```diff
@@ -122,4 +127,89 @@ library MaxInvocationsLib {
             }
         }
     }
+
+    /**
+     * Helper function to check if max invocations has been initialized.
+     * @dev This function checks if max invocations has been initialized.
+     * @param _maxInvocationsProjectConfig Struct that holds max invocations
+     * project configuration.
+     * @return bool
+     * @dev We know a project's max invocations have never been initialized if
+     * both max invocations and maxHasBeenInvoked are still initial values.
+     * This is because if maxInvocations were ever set to zero,
+     * maxHasBeenInvoked would be set to true.
+     */
+    function maxInvocationsIsInitialized(
+        MaxInvocationsProjectConfig storage _maxInvocationsProjectConfig
+    ) internal view returns (bool) {
+        return
+            !(_maxInvocationsProjectConfig.maxInvocations == 0 &&
+                _maxInvocationsProjectConfig.maxHasBeenInvoked == false);
```

Comment:
Nit: but I think this is easier to read, and I think _should_ be more gas efficient as it can short circuit this way on the first condition

```suggestion
            _maxInvocationsProjectConfig.maxInvocations != 0 || _maxInvocationsProjectConfig.maxHasBeenInvoked;
```

Reply:
good call - thanks for the suggestion!